src/java.base/share/classes/jdk/internal/util/ByteArray.java

```diff
@@ -25,13 +25,12 @@
 
 package jdk.internal.util;
 
-import java.lang.invoke.MethodHandles;
-import java.lang.invoke.VarHandle;
-import java.nio.ByteOrder;
+import jdk.internal.misc.Unsafe;
+import jdk.internal.vm.annotation.ForceInline;
 
 /**
  * Utility methods for packing/unpacking primitive values in/out of byte arrays
- * using {@linkplain ByteOrder#BIG_ENDIAN big endian order} (aka. "network order").
+ * using {@linkplain java.nio.ByteOrder#BIG_ENDIAN big endian order} (aka. "network order").
  * <p>
  * All methods in this class will throw an {@linkplain NullPointerException} if {@code null} is
```

Comment:
Do the Unsafe methods throw NPE on null array argument?

Reply:
> Do the Unsafe methods throw NPE on null array argument?

No, but `arrayOffset` will throw.